code/node_modules/relationships_doc.js

```diff
@@ -7,22 +7,20 @@ try {
     throw new Error('DataSource codebase not found. This application is designed to nest within an existing X-DataSource repo.') 
 }
 
-const SOURCEDIR = '/code/docs/source'
-const RELFILE = `${SOURCEDIR}/Relationships.rst`
```

Comment:
SOURCEDIR has been moved to `update.js` so that it can be modified outside of class definitions (if the need arises)